telegram_packs/other.txt

```diff
@@ -0,0 +1,88 @@
+other/biFrog|🏳️
+other/croakaColaFrogCan|🧃
+other/doomFrog|🔫
+other/dumbFrog|🤪
+other/eatenFrog|🪦
+other/fingeringFrog|👉
+other/frog2b|🤖
+other/frogBag|💰
+other/frogBatsignal|🌃
+other/frogBeerCan|🍺
+other/frogBeer|🍻🍺
+other/frogBender|🤖
+other/frogBlanket|🛌
+other/frogBlunt|🚬
+other/frogCatCostume|🐱
+other/frogCat|🐱
+other/frogChampagne|🍾
+other/frogChronoTrigger|🐲
```

Comment:
:video_game: ? cant think of a dragon association with CT...

Reply:
but the CT character is a dragon like frog right?

Reply:
added the controller as well

Reply:
![image](https://github.com/Riesi/frog_emojis/assets/27691485/43ddb0a1-33a2-4133-84f5-d5752717109e)
looks enough like a dragon to me

Reply:
![grafik](https://github.com/Riesi/frog_emojis/assets/38186597/70b87aee-7f2f-42e5-b417-c7f81e16e713)

It's a frog, not a dragon :(